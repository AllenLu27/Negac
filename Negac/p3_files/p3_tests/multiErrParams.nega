int fn(int a, void a){ }
