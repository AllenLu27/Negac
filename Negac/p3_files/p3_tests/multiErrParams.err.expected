FATAL [1,20]: Multiply declared identifier
FATAL [1,20]: Invalid type in declaration
Semantic analysis failed
